templates/project-ts/package.json

```diff
@@ -26,6 +32,7 @@
   },
   "lint-staged": {
     "**/*": [
+      "./node_modules/.bin/eslint src/* --fix",
```

Comment:
Do you know if this works with just `eslint`, similar to how prettier is below? 

When @MartinMinkov tested on Windows, I recall some issue with `.bin`

Reply:
I will double check this also